BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -300,12 +300,13 @@ private extension ConvivaAnalytics {
         }
 
         setupPlayerStateManager()
-        updateSession()
 
         videoAnalytics.reportPlaybackRequested(contentMetadataBuilder.build())
         logger.debugLog(message: "Creating session with metadata: \(contentMetadataBuilder)")
         logger.debugLog(message: "Session started")
         isSessionActive = true
+
+        updateSession()
```

Comment:
There is an early return within the `updateSession` method checking against the `isSessionActive` flag. Therefore the old position of the method call never did anything. I looked at old PRs where this was introduced and there we always had the `updateSession` call from within the `internalInitializeSession` so I decided to move it here to enable this behaviour again.